src/caches/abstractbeliefpropagationcache.jl

```diff
@@ -284,15 +315,14 @@ Do parallel updates between groups of edges of all message tensors
 Currently we send the full message tensor data struct to update for each edge_group. But really we only need the
 mts relevant to that group.
 """
-function update(
+function update_iteration(
   alg::Algorithm,
```

Comment:
```suggestion
  alg::Algorithm"bp",
```
and for the other `update_iteration` functions. Does that make sense to you?

Reply:
Yeah it does, have done that